app/views/users/dashboard.html.erb

```diff
@@ -16,10 +16,9 @@
           <p><%= booking[:payed] ? 'yes' : 'no' %></p>
         </div>
 
-        <div class= "dog-owner">
-        <%= link_to "Go to your dog's page", dog_path(@dogs[i][:id]), class: "links-sessions" %>
-        <%= link_to "Renter's profile", user_path(@renter[i][:id]), class:"btd-yel" %>
```

Comment:
No more link to renter's profile page ?
(I know it is not functional yet due to pundit issues but it would be good to still have it there in the meantime)

Reply:
well i was just thinking that i would rather not have it in here while it does not function , adding it afterwards would be a piece of cake , just a copy paste. BTW I was wondering, is it really usefull to have a direct link to the "renter" here, would it not be cooler to have a "message the owner" feature instead ? (well maybe for later)

Reply:
Then comment it out is easier.